docs/operator/test.md

```diff
@@ -0,0 +1,96 @@
+# Test guide
+
+---
+This document facilitates an example of working commands to run two networks on a set of drand daemons. Nodes will be created using `docker-compose`.
+The file can be found on drand repo, under `<project-folder>/test/docker*utils/docker-compose.yml`.
+
+## Pre requisites
+
+---
+1) Build docker images on drand repository as follows:
+
+```bash
+docker build --build-arg gitCommit=`git rev-parse HEAD` -t drandorg/go-drand:latest .
+```
+
+2) Start drand nodes as follows:
+
+```bash
+docker-compose up -d
```

Comment:
Sorry @emmanuelm41 I messed up the line: I meant this line. Starting the drand node shouldn't be hidden behind a docker compose file.

Reply:
I think it would be a mess to set those commands here. That is the purpose of docker compose... to make it easier and simpler to run containers. Each container has many arguments. I don't think it is hard for the user to see them directly from the file. What do you think?

Reply:
Yes, I like that idea. I will add it!